cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -748,32 +748,35 @@ def _add_efs_mount_target(
             checked_availability_zones.append(availability_zone)
 
     def _add_raid_volume(self, id: str, shared_ebs: SharedEbs):
-        """Add specific Cfn Resources to map the EBS storage."""
-        # TODO: add RAID storage
-        return f"fake_id{id}-{shared_ebs.mount_dir}"
+        """Add specific Cfn Resources to map the RAID EBS storage."""
+        ebs_ids = []
+        for _ in range(shared_ebs.raid.number_of_volumes):
+            ebs_id = self._add_cfn_volume(id, shared_ebs).ref
```

Comment:
Let's return the .ref from the _add_cfn_volume.

Reply:
Done